powersimdata/input/converter/pypsa_to_grid.py

```diff
@@ -3,10 +3,11 @@
 import numpy as np
 import pandas as pd
 
-from powersimdata.input.abstract_grid import AbstractGrid
+from powersimdata.input.abstract_grid import AbstractGrid, storage_template
```

Comment:
I believe the storage template is accessible from the `AbstractGrid` object:
```
>>> from powersimdata.input.abstract_grid import AbstractGrid
>>> ag = AbstractGrid()
>>> ag.storage
{'gen': Empty DataFrame
Columns: [bus_id, Pg, Qg, Qmax, Qmin, Vg, mBase, status, Pmax, Pmin, Pc1, Pc2, Qc1min, Qc1max, Qc2min, Qc2max, ramp_agc, ramp_10, ramp_30, ramp_q, apf, mu_Pmax, mu_Pmin, mu_Qmax, mu_Qmin]
Index: [], 'gencost': Empty DataFrame
Columns: [type, startup, shutdown, n, c2, c1, c0]
Index: [], 'StorageData': Empty DataFrame
Columns: [UnitIdx, InitialStorage, InitialStorageLowerBound, InitialStorageUpperBound, InitialStorageCost, TerminalStoragePrice, MinStorageLevel, MaxStorageLevel, OutEff, InEff, LossFactor, rho, ExpectedTerminalStorageMax, ExpectedTerminalStorageMin]
Index: [], 'genfuel': [], 'duration': None, 'min_stor': None, 'max_stor': None, 'InEff': None, 'OutEff': None, 'LossFactor': None, 'energy_price': None, 'terminal_min': None, 'terminal_max': None}
>>> 
```
and there is no need to import `storage_template`